tests/checkpointing/test_model_checkpoint.py

```diff
@@ -19,6 +19,28 @@
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 
 
+@pytest.mark.parametrize('save_top_k', [1])
+def test_model_checkpoint_correct_score(tmpdir, save_top_k):
```

Comment:
can we rather save all `save_top_k=-1` and check all... to be more robust :]

Reply:
Thanks! Should be fixed, could I get a recheck?